papers/whitepaper.md

```diff
@@ -0,0 +1,54 @@
+
+That said, there will be demand for Atoma Network functionality beyond smart contracts, directly. Applications such as UI chat applications, AI Web3 wallet integrations, online data scrapping,
+next generation browsers and browser extensions, community based knowledge bases, social media platforms, etc. Such applications will not require the same level of trust assumptions as general smart contracts, but will still require certain integrity guarantees. As an example, a DeFi protocol may require almost 100% certainty that a given AI market liquidity prediction was computed correctly, even a small deviation from the original prediction can have a huge impact on the protocol's future funds management. However, a user interacting with an online community based chat application is mostly interested in knowing that it paid the right fee to have access to the right model for output generation, and not a less capable model. Therefore, our sampling consensus is adapted to each of such use cases, being therefore highly flexible and optimizing for a wide range of different use cases. We are currently exploring other verifiability techniques, such as Trusted Execution Environments (TEEs), that will allow for even higher levels of verifiability, while also reducing the cost of verifiable execution close to native cost.
+
+The Atoma Network will be empowered by its TOMA token. Through a robust tokenomics, both compute providers, protocol participants and developers will be rewarded by their contribution to maintaining the network longevity. The TOMA token will be required to pay for AI inference fees, for nodes to deposit collateral to be able to participate on the network, and also for developers to being paid for their contributions to the Atoma Network. Moreover, the tokenomics follow an initial period of inflation in order to subsidize the network's growth. This growth includes both incentivizing node operators to acquire better compute hardware, to incentivize model creators to develop new AI models, specialized for applications of interest to the Atoma Network (such as financial market predictions, etc), as well as to incentivize developers to build applications that drive demand for the available compute resources on the Atoma Network. After the network's inflationary period (which will last a few years), the tokenomics will follow a more balanced dynamics, through tail emissions and burning mechanisms. Together, both mechanisms will lead to a gradual decrease in the total supply of the TOMA token, contributing for its increase in value provided the demand for Atoma Network increases, while also allowing for a some new allocation of the token to the protocol's treasury and to value providers (in the form of new node providers, model creators, developers, etc).
+
```

Comment:
The Atoma Network will be powered by its TOMA token. Through robust tokenomics, compute providers, protocol participants, and developers will be rewarded by their contribution to maintaining the network longevity. The TOMA token will be required to pay for AI inference fees, for nodes to deposit collateral to be able to participate on the network, and also for developers to be paid for their contributions to the Atoma Network.

This growth (of the network?) incentivizes node operators to acquire better compute hardware, and model creators to develop new AI models specialized for applications such as financial market predictions. It also incentivizes developers to build applications that drive demand for the available compute resources on the Atoma Network. 

The tokenomics follow an initial period of inflation in order to subsidize the network's growth. After the network's inflationary period (which will last a few years), the tokenomics will follow a more balanced dynamics, through tail emissions and burning mechanisms.

Together, both mechanisms will lead to a gradual decrease in the total supply of the TOMA token. TOMA will increase in value provided the demand for Atoma Network increases, while also allowing for some new allocation of the token to the protocol's treasury and to value providers (in the form of new node providers, model creators, developers, etc)

Reply:
Already refactored